examples/overviewmap/overview.html

```diff
@@ -1,44 +1,41 @@
 <!DOCTYPE html>
 <html>
-<head>
-    <title>GeoExt.OverviewMap Example</title>
+    <head>
+        <title>GeoExt.OverviewMap Example</title>
 
-    <!-- ExtJS -->
-    <script type="text/javascript" src="http://cdn.sencha.com/ext/gpl/4.2.1/examples/shared/include-ext.js"></script>
-    <script type="text/javascript" src="http://cdn.sencha.com/ext/gpl/4.2.1/examples/shared/options-toolbar.js"></script>
+        <!-- ExtJS -->
+        <script type="text/javascript" src="../include-ext.js"></script>
+        <script type="text/javascript" src="../options-toolbar.js"></script>
 
-    <!-- Shared -->
-    <link rel="stylesheet" type="text/css" href="http://cdn.sencha.com/ext/gpl/4.2.1/examples/shared/example.css" />
-    <!-- Basic example styling -->
-    <link rel="stylesheet" type="text/css" href="../shared/example.css" />
+        <!-- Basic example styling -->
+        <link rel="stylesheet" type="text/css" href="../shared/example.css" />
 
-    <!-- You should definitely consider using a custom single-file version of OpenLayers -->
-    <script src="http://openlayers.org/api/2.13.1/OpenLayers.js"></script>
-    <link rel="stylesheet" type="text/css" href="http://openlayers.org/api/2.13.1/theme/default/style.css">
```

Comment:
wondering why this was removed everywhere, because we don't construct our Maps with theme: null OpenLayers will try to load the css on demand? Or is this not the case?